README.md

```diff
@@ -4,70 +4,26 @@
-
-To include a figure or image in an article: 
- - save your figure or image as a `.svg` file, naming the file using the pattern `[article filename]-[figure name].svg`
- - add the figure/image to the **images** folder in this repo
-
-### Standard formatting for operators and operator properties
-
-When referring to operators (also known as nodes in Bonsai), place operator names inside a pair of backticks and double stars (``**`Operator_name`**``) so that the node names render as bold text in code snippet formatting (i.e. **`CameraCapture`**). 
-
-When referring to operator properties, simply place the operator property name inside a pair of backticks (`` `Operator_property_name` ``). 
-
-For both operators and operator properties, link the name to the relevant documentation in the code base, using the [markdown syntax for xref in DocFX](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html). 
-
-For example, the **`DigitalOutput`** node is part of the `Bonsai.Arduino` namespace/package, and contains a property called `PortName`. To reference this specific property you need to specify the full path to it including namespace, operator name and property name, like so: `xref:Bonsai.Arduino.DigitalOutput.PortName`. 
-
-To find out the full path for any node, right-click on the operator of interest in Bonsai and select the option "Go to Definition" or hit F12. The xref will be the namespace + operator name + (optionally) property name.
-
-**Example:**
-
-```markdown
-### **Exercise 7:** Control an LED
-
-![Control an LED](~/images/acquisition-led.svg)
-
-- Insert a [**`Boolean`**](xref:Bonsai.Expressions.BooleanProperty) source.
-- Insert a [**`DigitalOutput`**](xref:Bonsai.Arduino.DigitalOutput) sink.
-- Set the [`Pin`](xref:Bonsai.Arduino.DigitalOutput.Pin) property of the [**`DigitalOutput`**](xref:Bonsai.Arduino.DigitalOutput) operator to 13.
-- Configure the [`PortName`](xref:Bonsai.Arduino.DigitalOutput.PortName) property.
-- Run the workflow and change the [`Value`](xref:Bonsai.Expressions.BooleanProperty.Value) property of the [**`Boolean`**](xref:Bonsai.Expressions.BooleanProperty) operator.
-- **Optional:** Use your mouse to control the LED! Replace the [**`Boolean`**](xref:Bonsai.Expressions.BooleanProperty) operator by a `MouseMove` source (hint: use `GreaterThan`, `LessThan`, or equivalent operators to connect one of the mouse axis to [**`DigitalOutput`**](xref:Bonsai.Arduino.DigitalOutput).
+1. Fork the repository you want to contribute documentation to. 
+2. Ensure that you are working off the `main` branch in your fork and create a descriptively named branch for each article/issue that you want to work on.
+3. Download and install dotnet (https://dotnet.microsoft.com/en-us/download)
+4. Restore `docfx` by running
+```powershell
+dotnet tool restore
 ```
+5. In a Windows Powershell, use the command `dotnet docfx docfx.json --serve` to generate a local preview of the documentation website as you are making changes.
```

Comment:
So I was debating what to keep from the original README and how to structure the content to avoid duplication.

I think we should keep the README short and simple, as people who are looking to make contributions to documentation might not need to know the ins and outs of docfx as documented in the `documentation-docfx.md` or even maybe the style guidelines in `documentation-style-guide.md` (if they just follow the formatting of existing articles). It looks less intimidating, contains just the minimum necessary information and lowers the barrier to entry for contributions. And then we have links to those who want to explore more, need more guidance or face problems with docfx. 

I think the stepping into `docs` folder is a good point though and I will add it because the main docs website is structured differently.